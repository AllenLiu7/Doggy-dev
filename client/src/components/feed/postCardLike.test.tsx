import { shallow } from 'enzyme';
import React from 'react';

import PostCardLike from './postCardLike';

describe('PostCardLike component', () => {
  test('renders', () => {
    const wrapper = shallow(
      <PostCardLike handleLikeClick={() => {}} like={1} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
