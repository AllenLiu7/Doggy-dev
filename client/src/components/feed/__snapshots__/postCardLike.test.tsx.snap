// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostCardLike component renders 1`] = `
<styled.div>
  <styled.img
    onClick={[Function]}
  />
  <styled.img />
  <styled.div>
    1 people likes it
  </styled.div>
</styled.div>
`;
